norduser/utmp.go

```diff
@@ -0,0 +1,94 @@
+package norduser
+
+/*
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include <utmp.h>
```

Comment:
At least according to the documentation functions in this header are obsolete and new applications should use `<utmpx.h>`, `getutxent()` and `endutxent()`: https://man7.org/linux/man-pages/man3/getutid.3.html